components/EnterStateName.tsx

```diff
@@ -0,0 +1,9 @@
+import {Text, View, Image} from 'react-native';
+
+export default function EnterStateName(){
+    return(
+        <View>
+            <Text>Enter the state name</Text>
+        </View>
+    )
```

Comment:
These files feel like individual components rather than an entire page. I do not understand what exactly do you mean by a file called `EnterStateName`. That looks more like a prompt you would display. Label for a text field or something.

Reply:
Its basically aimed to let the user type the state's name manually, particularly aimed to improve the user's spelling knowledge. Its a blank text field wherein the user will have to enter the state's name on which the input text field is located.